### DISCUSSION TITLE
Any reports of this actually working?

### DISCUSSION THREAD

**Discussion by CheeseTheWeese** [GiorgioBrux/nitro-sniper-enhanced#131] [Q&A]
has anyone ever gotten dropped nitro from this?

**Comment by CheeseTheWeese** [comment]
i keep getting "already redeemed" codes. isnt a nitro sniper supposed to be faster than humans? can anyone confirm that this thing isnt just taking nitro and sending it to the creators acc or sum

**Reply by GiorgioBrux** [reply]
FYI it's FOSS for a reason. It 100% works, you can test this by sending yourself a gift from another account in DMs and you'll see it auto-redeemed.
